sdk/ts/README.md

```diff
@@ -0,0 +1,33 @@
+# Ursa Typescript SDK
+
+The typescript sdk provides an easy interface for interacting with the Ursa network.
+
+## Development
+
+### Prerequisites
+
+-   [npm](https://www.npmjs.com/get-npm)
+
+### Setup
+
+```bash
+npm i
```

Comment:
can we use yarn instead

Reply:
yeah https://github.com/fleek-network/ursa/pull/370/commits/b60f4897397eee3742072da86705d03f309e35d7